### DISCUSSION TITLE
When generating video and adding interpolation steps, Video is not saved?

### DISCUSSION THREAD

**Discussion by Bwebbfx** [painebenjamin/app.enfugue.ai#99] [Q&A]
Hey, everything works really well and is a slick interface, was wondering if this behavior is expected, I have also had videos not save when not using interpolator, in both cases, they still show on the canvas and can play, is there a way to force a video export? thanks!

**Comment by painebenjamin** [comment]
Hello @Bwebbfx, thank you for the report!

Thank you for the kind words :) I made sure to save videos _prior_ to interpolation in case the interpolation fails (the FILM interpolator is more finnicky than I wish it was, I'm looking to replace it.) You should be able to see your video in `File > Results`. The "Output" column will have a thumbnail version of the video, if you click it you will open a new tab to the `.mp4` file on it's own. You can also click `view as .gif` to convert to a gif, or click "View Frame" to place the animation back into the scrubber.

![image](https://github.com/painebenjamin/app.enfugue.ai/assets/57536852/2ebeafd1-c4a0-4543-8a6c-ec5824aef154)

*If you don't see this*, it's possible your installation is having difficulties with codecs, and enfugue can't write .mp4 files for some reason. You'll know if this is the case if you try and make an animation with or without interpolation and enfugue says something like `Nothing was written to /opt/rd/enfugue/images/69f0338f3f014f1685a275183f7e317e.mp4`, and you'd likely find an error in the logs along the lines of `[ERROR:0@468.923] global cap_ffmpeg_impl.hpp:3018 open Could not find encoder for codec_id=27, error: Encoder not found`. If this is the case I'd love a bit more information about your system and how you installed enfugue - and whether or not you have other software that works in decoding/encoding video. In theory I should be installing FFMPEG and OpenCV which will give me the necessary tools to read and write `.mp4` files, but in practice there could be environment conflicts that I'm not aware of - the solution to this would be to make sure the codecs are available on your system via installing OpenCV. I don't want to lead you down that path if it doesn't need to happen though, so please let me know if you find anything in the logs that jumps out to you.

Thanks, and sorry again for the hassle!

**Comment by Bwebbfx** [comment]
Awesome, thank you so much, was able to access the generations i thought i lost! maybe an addition prompt after generation with interp to try to write it out? either way, works great thanks!
